### ISSUE TITLE
"Loud" style Comment throws "silent" message

### ISSUE THREAD

**Issue by meowsus** [brigade/scss-lint#865]
In my .scss-lint.yml config I've enabled "loud" style comments:

```yml
linters:
  Comment:
    style: loud
```

When I run this on my codebase, when I hit the following offense:

```scss
// test comment
```

I receive the following message:

```
Use `//` comments everywhere
```

I believe this message should actually read:

```
Use `/* */` comments everywhere
```

Or something along those lines.
